test/MVCFramework20.FunctionalTests/FunctionalTest/TelemetryModuleWorkingMvcTests.cs

```diff
@@ -17,7 +18,18 @@ public TelemetryModuleWorkingMvcTests(ITestOutputHelper output) : base(output)
         [Fact]
         public void TestBasicDependencyPropertiesAfterRequestingBasicPage()
         {
-            this.ValidateBasicDependency(assemblyName, "/Home/About/5", InProcessServer.UseApplicationInsights);
+            const string RequestPath = "/Home/About/5";
+
+            using (var server = new InProcessServer(assemblyName))
```

Comment:
Is there a reason of moving this line (using (var server = new InProcessServer(assemblyName))) from base class to each of the subclasses?

Reply:
It was in base class only for dependencies, but for requests and exceptions, it was out of it in individual cases - i moved it out of here for dependencies as well to be more consistent in all methods.